### PR TITLE
Upgrade to Android API 29  + gradle 6 +security fix

### PR DESCRIPTION
**Pull request by ralic**
Tested on my machine (windows 10/node.js 13.3.0) , running react-native 0.61.4

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,11 +26,7 @@
     "ios",
     "android"
   ],
-  "author": {
-    "name": "Andy Prock",
-    "email": "aprock@gmail.com"
-  },
-  "license": "MIT",
+  "license": "MIT"
```
Comment (majorhayes) [inline package.json:29]: Missing comma

Comment (ralic) [reply package.json]: Done

**Comment by joeyfigaro** [comment]
@aprock ping

**Review by shairamiri — APPROVED** [review]

**Comment by vietnamese-chowder** [comment]
Why has this not been merged? @aprock

**Comment by Moistbobo** [comment]
As this update has not been uploaded to npm yet, you will have to point your `package.json` to either use this merge commit, or the latest commit on master.

In `package.json` dependencies:

Use this specific merge commit
`"react-native-os": "aprock/react-native-os.git#938676ebd92d58b8928168a7e78cbe216838ed17"`

Use the latest commit on master
`"react-native-os": "aprock/react-native-os.git#master",`

**Comment by Schoonmoeder** [comment]
my pod project ,for using this in iOS app, dislikes that the author was removed from the package.json
can this me returned?
```
"author": {
    "name": "Andy Prock",
    "email": "aprock@gmail.com"
  },
```

**Comment by Schoonmoeder** [comment]
Missing author in package.json gives error when building in POD (for ios)
please add it again
```
"author": {
    "name": "Andy Prock",
    "email": "aprock@gmail.com"
  }, 
```

**Comment by dac245** [comment]
@aprock Any chance this commit can be versioned? Would be helpful with the build.gradle changes.

**Pull request merged**